arwen/elrondapi/elrondei.go

```diff
@@ -2271,6 +2435,7 @@ func prepareIndirectContractCallInput(
 	numArguments int32,
 	argumentsLengthOffset int32,
 	dataOffset int32,
+	syncExecution bool,
```

Comment:
Suggest renaming this parameter to `syncExecutionRequired`, it appears to be a better name given its usage in the `if` below.

Reply:
done